stage1/init/pod.go

```diff
@@ -131,6 +131,33 @@ func quoteExec(exec []string) string {
 	return strings.Join(qexec, " ")
 }
 
+func (p *Pod) WriteDefaultTarget() error {
+	opts := []*unit.UnitOption{
+		unit.NewUnitOption("Unit", "Description", "rkt apps target"),
+		unit.NewUnitOption("Unit", "DefaultDependencies", "false"),
+	}
+
+	for i := range p.Manifest.Apps {
+		ra := &p.Manifest.Apps[i]
+		serviceName := ServiceUnitName(ra.Name)
+		opts = append(opts, unit.NewUnitOption("Unit", "After", serviceName))
+		opts = append(opts, unit.NewUnitOption("Unit", "Wants", serviceName))
+	}
+
+	unitsPath := filepath.Join(common.Stage1RootfsPath(p.Root), unitsDir)
+	file, err := os.OpenFile(filepath.Join(unitsPath, "default.target"), os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	if err != nil {
+		return fmt.Errorf("failed to create service unit file: %v", err)
```

Comment:
Given that the caller also builds up an error message, this would print something like:
```
Failed to write default.target: failed to create service unit file: Permission denied
```
Can it be made shorter? Also, it's not a service unit file but a target..